src/os/config/displaysettings.js

```diff
@@ -70,6 +71,15 @@ os.config.isTerrainConfigured = function() {
   return !!(options && options.type);
 };
 
+/**
+ * If fog has been configured in the application.
+ *
+ * @return {boolean}
+ */
+os.config.isFogConfigured = function() {
+  var fogSupport = /** @type {boolean|undefined} */ (os.settings.get(os.config.DisplaySetting.FOG_UNSUPPORTED));
```

Comment:
For clarity, this should probably provide a default value if this key is unconfigured: `os.settings.get(SOME_KEY, false)` in this case, or `os.settings.get(FOG_SUPPORTED, true)` if the name is changed to positive.